collectors/ebpf.plugin/ebpf.h

```diff
@@ -98,6 +99,26 @@ typedef struct netdata_error_report {
     int err;
 } netdata_error_report_t;
 
+typedef struct netdata_ebpf_judy_pid {
+    ARAL *pid_table;
+
+    // Index for PIDs
+    struct {                            // support for multiple indexing engines
+        Pvoid_t JudyHSArray;            // the hash table
+        RW_SPINLOCK rw_spinlock;        // protect the index
+    } index;
+} netdata_ebpf_judy_pid_t;
+
+typedef struct netdata_ebpf_judy_pid_stats {
+    char *cmdline;
+
+    // Index for Socket timestamp
+    struct {                            // support for multiple indexing engines
+        Pvoid_t JudyHSArray;            // the hash table
```

Comment:
Same here. This is actually a judyL array. Although its just the name of the variable, if you can, change it to avoid confusion. eg JudyLArray or just JudyL

Reply:
The same comment above.